ci/docker/runtime_functions.sh

```diff
@@ -73,6 +73,27 @@ build_jetson() {
     popd
 }
 
+build_armv6() {
+    set -ex
+    pushd .
+    cd /work/build
+    cmake\
+        -DUSE_CUDA=OFF\
+        -DUSE_OPENCV=OFF\
+        -DUSE_OPENMP=OFF\
+        -DUSE_SIGNAL_HANDLER=ON\
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo\
+        -DUSE_MKL_IF_AVAILABLE=OFF\
+        -DUSE_LAPACK=OFF\
```

Comment:
According to https://github.com/apache/incubator-mxnet/blob/v1.1.0/docker_multiarch/Dockerfile.build.armv6, LAPACK used to be enabled - it would be good if we could keep it that way.